ocrd_utils/ocrd_utils/__init__.py

```diff
@@ -82,6 +207,19 @@ def is_string(val):
     """
     return isinstance(val, str)
 
+def membername(class_, val):
+    """Convert a member variable/constant into a member name string."""
+    return next((k for k, v in class_.__dict__.items() if v == val), str(val))
+
+def points_from_bbox(minx, miny, maxx, maxy):
+    """Construct polygon coordinates in page representation from a numeric list representing a bounding box."""
+    return "%i,%i %i,%i %i,%i %i,%i" % (
+        minx, miny, maxx, miny, maxx, maxy, minx, maxy)
+
+def points_from_polygon(polygon):
+    """Convert polygon coordinates from a numeric list representation to a page representation."""
+    return " ".join("%i,%i" % (x, y) for x, y in polygon)
+
 def points_from_xywh(box):
     """
     Constructs a polygon representation from a rectangle described as a dict with keys x, y, w, h.
```

Comment:
Please use the same system for describing the converter docstrings in the old functions as well (making them more precise and consistent with the new ones).
```suggestion
    Construct polygon coordinates in page representation from numeric dict representing a bounding box.
```